helm/reana/templates/database-config.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
Hmm, I wonder whether it would be possible to amend the setup to automatically delete / refresh `reana-server` and `reana-workflow-controller` pods when the admin changes PgBouncer from enabled to disabled and vice versa? (Currently in addition to `helm diff upgrade...` one has to do it manually.) 

This comment is a sort of a "bonus", the things work nicely :+1: and I'm not sure whether admins would attempt to play with enabling/disabling PgBouncer dynamically in reality... Well we could always simply explain this in the documentation.

Reply:
I have updated the PR to also roll the deployments of pgbouncer/r-server/r-w-controller when the database config (configmap or secrets) change. This can be used as a blueprint to do the same for the other config/secret that might change over time.